src/js/components/App.test.js

```diff
@@ -0,0 +1,52 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import App from './App';
+import helpers from '../helpers';
+
+describe('App', () => {
+  it('should be defined', () => {
+    expect(App).toBeDefined();
+  });
+
+  it('should render correctly', () => {
+    const wrapper = shallow(<App />);
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should emit search when doSearch emmit', () => {
+    const wrapper = shallow(<App />);
+    wrapper.instance().sendQuery = jest.fn();
+    wrapper.instance().doSearch();
+    expect(wrapper.instance().sendQuery).toHaveBeenCalledTimes(1);
```

Comment:
It looks like we are testing implementation here, which we shouldn't be doing. I would mock helpers module and and verify that `helpers.getData` has been called with specified arguments, for example